dali/dfuxref/dfuxrefmain.cpp

```diff
@@ -50,12 +50,17 @@ int main(int argc, char* argv[])
     setNodeCaching(true);
 
     {
+#ifndef _CONTAINERIZED
         Owned<IComponentLogFileCreator> lf = createComponentLogFileCreator("dfuxref");
         lf->setAppend(false);
         lf->setMaxDetail(TopDetail);
         lf->setMsgFields(MSGFIELD_STANDARD);
         lf->beginLogging();
+#else
+        updateContainerizedLogMsgHandler(getCategoryLogMsgFilter(MSGAUD_all, MSGCLS_all, TopDetail));
```

Comment:
ditto to above, and it's doubtful XREF  ever will be in cloud/containerized env.